x-pack/plugins/security_solution/public/management/common/translations.ts

```diff
@@ -40,3 +41,10 @@ export const OS_TITLES: Readonly<{ [K in OperatingSystem]: string }> = {
     defaultMessage: 'Linux',
   }),
 };
+
+export const getLoadPoliciesError = (error: ServerApiError) => {
+  return i18n.translate('xpack.securitySolution.hostIsolationExceptions.failedLoadPolicies', {
```

Comment:
Is the translation id ok to be `hostIsolationException`? or should it be something more generic, since it is in common?